src/main/java/ca/intelliware/ihtsdo/mlds/web/rest/UserNotifier.java

```diff
@@ -16,11 +18,14 @@ public class UserNotifier {
 	@Resource ReleasePackageUpdatedEmailSender releasePackageUpdatedEmailSender;
 	@Resource UserMembershipCalculator userMembershipCalculator;
 
+    private final Logger log = LoggerFactory.getLogger(UserResource.class);
+
 	public void notifyReleasePackageUpdated(ReleaseVersion releaseVersion) {
```

Comment:
@markrynbeek What distinction is being made here between being the user being notified about an international release vs a country release?   So I can see that if this is a member and they've said they only want country notification then this email will not be sent, but what if this WAS actually a country release?   Is there a separate method for that or do we need to work out what sort of release package we're looking at?   Thanks!

Reply:
Hi @pgwilliams I thought the test on member.getKey() would work for that. Key 'IHTSDO' indicating an international release, and other key (like AT, Austria) indicating a country release?

Reply:
Yes it does, thanks that answers my question.   I'll approve and pull now and we can see how it runs in Dev.   Did you modify the email addresses?

Reply:
I did modify the non IHTSDO.org email addresses that had no "pwi+" prefix (it was a bit of a mixed bag). I've set acceptNotifications to false in the dev DB for almost all users too.